dts/arm/st/u5/stm32u5f9.dtsi

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2025 Charles Dias
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <st/u5/stm32u599.dtsi>
+
+/ {
+	soc {
+		compatible = "st,stm32u5f9", "st,stm32u5", "simple-bus";
+	};
```

Comment:
Missed that it would maybe be worth to consider also the SRAM6 (a diff between u59*/u5A* and u5F*/u5G*) in the `sram0` node?
Doing so would overwrite node sram0 from **stm32u599.dtsi**.